GOVERNANCE.rst

```diff
@@ -27,18 +27,20 @@ Authorship and maintainership of algorithms
 
-Algorithms are encapsulated in classes. To faciliate authorship and maintainership questions, each algorithm is written in a separate file.
+In sktime, algorithms are encapsulated in classes and are called estimators. Authorship and maintainership operate on the level of classes. To faciliate authorship and maintainership questions, we try to write algorithms in separate files when possible. If an algorithm uses multiple files, we try to collect them in a folder.
 
 The author is the contributor who implements and contributes the algorithm to the project. The maintainer is the contributor who is responsible for maintaining an algorithm and the first point of contact for users and other contributors for all issues, questions, and proposals regarding their algorithm. The author automatically becomes the first maintainer of the algorithm.
 
 * **Veto right.** The maintainer has the right to veto any proposed changes to their algorithm within a time frame of 1 week. This does not extend to change to the common framework and API.
 
-* **Maintenance duty.** The maintainer is responsible for maintaining an algorithm. This includes understandable and instructive documentation, bugs fixes, unit testing, coding quality and style improvements, and compliance with sktime's common API. Maintainers should reply to a new issue on their algorithm within 1 week, and demonstrate a significant effort to fix any issue within 1 month.
+* **Maintenance duty.** The maintainer is responsible for maintaining an
+algorithm. This includes understandable and instructive documentation, bugs fixes, unit testing, coding quality and style improvements, and compliance with sktime's common API. Maintainers should reply to a new issue on their algorithm within 10 working days, and demonstrate a significant effort to fix any issue within 30 working days, ignoring any public holidays.
 
-Any maintainer is expected to give up their role if they can no longer fulfil their responsibilities. If a  maintainer fails to fulfil their responsibilities in given time frame, the maintainer loses their role and all rights that come with it.
+Any maintainer is expected to give up their role if they can no longer
+fulfil their maintenance duty. If a  maintainer fails to fulfil their
+responsibilities in the given time frame, the maintainer loses their role and all rights that come with it.
 
-Note that authors and maintainers do not *own* the algorithms or files they
-contributed or maintain. sktime is an open-source project, and all code is contributed under our 3-clause-BSD license. By contributing to sktime, ownership is transferred to all sktime developers and the project as a whole.
+Note that authors and maintainers do not *own* the algorithms or files they contributed or maintain. sktime is an open-source project, and all code is contributed under our 3-clause-BSD license. By contributing to sktime, ownership is transferred to all sktime developers and the project as a whole.
 
```

Comment:
this should perhaps be directly linked to in PR template - i.e., acknowledge that this becomes subject to 3-clause-BSD, and that the author has sufficient rights to the code to commit it. If the code is not fully original, it should be subject to a license and form that makes contribution to sktime possible, and the committer needs to confirm that it is to the best of their knowledge.